### ISSUE TITLE
Accra, Ghana has orphaned descendants

### ISSUE THREAD

**Issue by henrylyne** [whosonfirst-data/whosonfirst-data#1828]
When I go to https://spelunker.whosonfirst.org/id/421168965/ I see:
OH NOES!!! MISSING LABEL CENTROID
Is that an indication of the problem here?
The neighborhood descendants of the locality Accra are missing a parent.

For example the Hierarchy for Nima https://spelunker.whosonfirst.org/id/85765569/
has "a locality that we aren't able to index correctly because... ?"

What's the problem for Nima and/or Accra? What's needed to fix?

Is reporting these issues helping?

**Comment by missinglink** [comment]
The associated `locality_id` is `-1` (which means unknown).

I'm guessing that it's because it's a neighborhood and so it should really belong to a locality.

I'm not sure why it's not just omitted rather than setting it -1?

**Comment by nvkelso** [comment]
Most likely it's because it's been a while since this record was first
imported, and the tooling has gotten better since then w/r/t common
properties and their values. If it was re-exportified it'd probably fix
itself to -1.

**Comment by henrylyne** [comment]
Yes, the parent/locality id of Nima is -1, perhaps because at some point Accra didn't have a polygon defined and Nima couldn't find it. Or perhaps because of some issue related to "MISSING LABEL CENTROID".

**Comment by stepps00** [comment]
Thanks @henrylyne.. some comments:

> perhaps because at some point Accra didn't have a polygon defined and Nima couldn't find it. 

Yeah, some capital city geometries were updated without updating their descendant records through https://github.com/whosonfirst-data/whosonfirst-data/issues/1496. There is follow up work to update capital city descendants, tracked here: https://github.com/whosonfirst-data/whosonfirst-data/issues/1549. Once complete, this would fix this issue and similar `-1` parent issues.

> Or perhaps because of some issue related to "MISSING LABEL CENTROID".

The `"MISSING LABEL CENTROID"` issue is separate, this comes from the record not having a `src:` property for the label centroid.. code here:
https://github.com/whosonfirst/whosonfirst-www-spelunker/blob/d9bb5ff6928e7ecc51d4ac3320fd80c0e6d9d833/www/templates/id.html#L119
